go/vt/vttablet/customrule/filecustomrule/filecustomrule.go

```diff
@@ -53,23 +53,35 @@ func NewFileCustomRule() (fcr *FileCustomRule) {
 	return fcr
 }
 
+// ParseRules will construct a Rules object based on a file path. On any error
+// nil and that error will be returned. A log will be printed to capture the
```

Comment:
This was confusing to me at first, in that I thought maybe you dropped some words. What do you think of:

> In the case of any error, it returns nil and that error.

Or maybe just adding a comma so "on any error nil" => "on any error, nil and that error"